be/src/exec/pipeline/operator.cpp

```diff
@@ -40,7 +40,7 @@ Operator::Operator(OperatorFactory* factory, int32_t id, const std::string& name
 }
 
 Status Operator::prepare(RuntimeState* state) {
-    _mem_tracker = state->instance_mem_tracker();
+    _mem_tracker = std::make_shared<MemTracker>(_common_metrics.get(), -1, _name, nullptr);
```

Comment:
The parent of `_mem_tracker` maybe should set to `state->instance_mem_tracker()`, not `nullptr`.
The `byte_limit` may be set to `query_options->mem_limit`, not -1.

Reply:
No, needn't. all the memory limit will be effected by instance's MemTracker by hook, here is only duplicate record just for profile analysis.

Reply:
> No, needn't. all the memory limit will be effected by instance's MemTracker by hook, here is only duplicate record just for profile analysis.

@liuyehcf It seems that the consumed memory is only recorded by `state->instance_mem_tracker()`, not by `Operator::_mem_tracker`. Will this lead to `_mem_tracker` always 0?